lucid/utils.py

```diff
@@ -74,19 +81,28 @@ def addWidget(self, widget):
                           grid_position[1])
 
 
-def indicator_for_device(device):
-    """Create a QWidget to indicate the alarm state of a QWidget"""
-    # This is a placeholder. There will be a system for determining the mapping
-    # of Device to icon put in place
-    circle = PyDMDrawingCircle()
-    circle.setStyleSheet('PyDMDrawingCircle '
-                         '{border: none; '
-                         ' background: transparent;'
-                         ' qproperty-penColor: black;'
-                         ' qproperty-penWidth: 2;'
-                         ' qproperty-penStyle: SolidLine;'
-                         ' qproperty-brush: rgba(82,101,244,255);} ')
-    return circle
+if TyphosAlarmCircle is not None:
+    def indicator_for_device(device):
+        """Create a QWidget to indicate the alarm state of a QWidget"""
+        circle = TyphosAlarmCircle()
```

Comment:
The only other thing I was considering was seeing how it looked/performed with:
```suggestion
        circle = TyphosAlarmCircle()
        circle.kindLevel = circle.KindLevel.NORMAL
```
Hypotheses:
- `hinted`, the default, might not catch all the issues with some devices
- `normal` might slow down the load

But I'm really not sure

Reply:
I was surprised at just how fast it loaded the sample hutches I tried with `normal`, to be honest.

That said, I agree that `hinted` may be sufficient enough and will almost certainly lead to even faster loading. Perhaps we leave it as-is and try it out a bit more, falling back to hinted if necessary?

Reply:
I opened all the lucids with hinted and with normal, hinted seems more friendly to me (smaller tooltips) and normal didn't reveal any new erroring devices. So let's leave it as-is I think